components/post_view/post_add_channel_member/post_add_channel_member.jsx

```diff
@@ -60,9 +60,11 @@ export default class PostAddChannelMember extends React.PureComponent {
         const {currentUser, post, userIds, usernames} = this.props;
 
         if (post && post.channel_id) {
+            let createAt = parseInt(post.create_at, 10);
```

Comment:
Why do you need to parse the timestamp here? Shouldn't `post.create_at` already be a number?

Reply:
ah, right... I'll update that.

Reply:
Thanks Harrison! It's now updated.